src/components/GirlsHooHack/Footer/Footer.tsx

```diff
@@ -1,8 +1,36 @@
 import React from 'react';
-
+import blueWaves from '../../../assets/images/footerWaveGHH.png';
+import whiteLogo from '../../../assets/images/whitelogo.png';
+import email from '../../../assets/images/envelope-solid.svg';
+import instagram from '../../../assets/images/instagram-square-brands.svg';
+import discord from '../../../assets/images/discord-brands.svg';
+import facebook from '../../../assets/images/facebook-f-brands.svg';
+import twitter from '../../../assets/images/twitter-brands.svg';
+import {Row} from 'react-bootstrap';
+import {isMobile} from 'react-device-detect';
+ 
 function Footer() {
   return (
-    <p>Footer</p>
+    <div>
+      <div className="bg-peacock" style={{width:"100%"}}>
+        <div className={isMobile ? "" : "pt-6 bg-white"}>
+          <div className={isMobile ? "ml-3" : "ml-4"}>
+            <img 
+              src={whiteLogo} className={!isMobile ? "mb-4" : ""} width={isMobile ? "125px" : "359px"} height={isMobile ? "39.7px" : "114px"} 
+              alt="logo" style={ isMobile ? {position:'relative', top:'75px'} : {position:'relative', zIndex: 100, top:'370px', left:'100px'} }
```

Comment:
why are you using style here? why can't do use padding or bootstrap to accomplish these goals?